src/firefly/js/templates/lightcurve/ztf/ZTFPlotRequests.js

```diff
@@ -62,7 +64,7 @@ export function getWebPlotRequestViaZTFIbe(tableModel, hlrow, cutoutSize, params
         let wp = null;
         sr.setParam('doCutout', 'false');
         if (!isNil(ra) && !isNil(dec)) {
-            sr.setParam('center', `${ra},${dec}`);
+            sr.setParam('center', `${cra},${cdec}`);
```

Comment:
Shouldn't be `${ra},, ${dec}` so it match 'sra,sdec' position?

Reply:
Oh, that's right. I was playing around with the cutout center. I will change it back.